spec/unit/validator_spec.rb

```diff
@@ -995,17 +998,31 @@ def test_swiftpod(swift_version = '3.1.0')
         Pod::Installer::PodSourceInstaller.any_instance.stubs(:unlock_files!)
         validator = Validator.new(file, config.sources_manager.master.map(&:url))
         validator.stubs(:build_pod)
-        validator.swift_version = swift_version
         validator.stubs(:validate_url)
         validator
       end
 
+      # Creates a test fixture using the `--swift-version` parameter.
+      def test_swiftpod_with_swift_version_parameter(swift_version = '3.1.0')
```

Comment:
re factored these methods out to be explicit on what kind of setup a validator has with regards to Swift version.